bin/composer

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec docker-compose run -it dev-container composer "$@"
```

Comment:
We could just use `docker run ...` and not require the changes to the compose file.